File: android/src/main/kotlin/com/giphy/giphy_flutter_sdk/GiphyFlutterSdkInfo.kt
package com.giphy.giphy_flutter_sdk

import android.content.Context
import com.giphyfluttersdk.BuildConfig

class GiphyFlutterSdkInfo(val context: Context) {
    val name = "FlutterSDK"

    val version: String
        get() {
            return BuildConfig.SDK_VERSION
        }
}

File: ios/giphy_flutter_sdk/Sources/giphy_flutter_sdk/dto/GiphyFlutterSdkInfo.swift
//
//  GiphyFlutterSdkInfo.swift
//  giphy_flutter_sdk
//
//  Created by Alex on 24.2.2024.
//

import Foundation

class GiphyFlutterSdkInfo {
    static let shared = GiphyFlutterSdkInfo()
    
    let name = "FlutterSDK"
    
    lazy var version: String? = {
        if let rawVersion = Bundle(for: type(of: self)).object(forInfoDictionaryKey: "CFBundleShortVersionString") {
            return rawVersion as? String
        }
        return nil
    }()
}
